origin-bridge/api/modules/attestations.py

```diff
@@ -1,10 +1,66 @@
 from flask import request
 from flask_restful import Resource
-from marshmallow import fields
+from marshmallow import Schema, fields
 from logic.attestation_service import VerificationService
 from api.helpers import StandardRequest, StandardResponse, handle_request
 
 
+class PubAuditableUrl(Schema):
+    challenge = fields.Str()
+    proofUrl = fields.Str()
+
+
+class VerificationMethod(Schema):
+    oAuth = fields.Boolean()
+    phone = fields.Boolean()
+    email = fields.Boolean()
+    mail = fields.Boolean()
+    human = fields.Boolean()
+    pubAuditableUrl = fields.Nested(PubAuditableUrl)
+
+
+class Hash(Schema):
+    function = fields.Str()
+    value = fields.Str()
+
+
+class Attribute(Schema):
+    raw = fields.Str()
+    verified = fields.Boolean()
+    hash = fields.Nested(Hash)
+
+
+class Site(Schema):
+    siteName = fields.Str()
+    userId = fields.Nested(Attribute)
+    username = fields.Nested(Attribute)
+    profileUrl = fields.Nested(Attribute)
+
+
+class Attestation(Schema):
+    verificationMethod = fields.Nested(VerificationMethod)
+    email = fields.Nested(Attribute)
+    phone = fields.Nested(Attribute)
+    site = fields.Nested(Site)
+
+
+class Issuer(Schema):
+    name = fields.Str()
+    url = fields.Str()
+    ethAddress = fields.Str()
+
+
+class AttestationData(Schema):
+    issuer = fields.Nested(Issuer)
+    issueDate = fields.Str()
+    attestation = fields.Nested(Attestation)
+
+
+class Signature(Schema):
+    bytes = fields.Str()
```

Comment:
Both `bytes` and `hash` are reserved words in Python. I don't believe it will cause an issue but generally I try to avoid using reserved words.

Reply:
Yes, that's good practice. Not causing an issue so will leave as is for now to keep code simple.